pkg/collectinfo/resources.go

```diff
@@ -11,30 +11,35 @@ import (
 const (
 	NodeKind              = "Node"
 	PVCKind               = "PersistentVolumeClaim"
+	PVKind                = "PersistentVolume"
 	STSKind               = "StatefulSet"
 	SCKind                = "StorageClass"
 	AerospikeClusterKind  = "AerospikeCluster"
 	PodKind               = "Pod"
 	EventKind             = "Event"
 	MutatingWebhookKind   = "MutatingWebhookConfiguration"
 	ValidatingWebhookKind = "ValidatingWebhookConfiguration"
+	ServiceKind           = "Service"
 )
 
 var (
 	KindDirNames = map[string]string{
 		NodeKind:              "nodes",
 		PVCKind:               "persistentvolumeclaims",
+		PVKind:                "persistentvolumes",
```

Comment:
Is it possible to filter out required or relevant PVs, maybe using CR storageClass and all?
As PV is a clusterscope resource, there can be 100s PV present in the cluster